src/modules/subject/subjects.routes.js

```diff
@@ -11,13 +11,35 @@ const {
 router.use(authMiddleware)
 router.param('id', idValidation)
 
+// @desc    Get all subjects
+// @route 	GET /subject
+// @access  Private (Authenticated users)
 router.get('/', asyncWrapper(subjectController.subjectsFind))
```

Comment:
is there any router for getting full list of subjects names? as it implemented for categories module